src/scripts/babel-gettext-extractor/index.js

```diff
@@ -223,6 +224,33 @@ export default function plugin() {
         }
 
         context[translate.msgid] = translate
+        if (
+          replacements &&
+          (typeof replacements[translate.msgid] === 'string' ||
+            typeof replacements[translate.msgid_plural] === 'string')
+        ) {
+          const newTranslate = {
+            ...translate,
+            comments: {
+              ...translate.comments,
+              extracted: (
+                (translate.comments.extracted || '').trim() + ` REPLACEMENT`
+              ).trim(),
```

Comment:
Do you think it make sense to have default comment in message.pot?
```suggestion
              extracted: (`REPLACEMENT for ${translate.msgid}`).trim(),
```

-> Otherwise message.pot may just have multiple empty comments like: 

```
#. REPLACEMENT
msgid ...
msgstr ...

#. REPLACEMENT
msgid ...
msgstr ...

#. REPLACEMENT
msgid ...
msgstr ...
```